examples/Podfile

```diff
@@ -26,14 +30,21 @@ target "EX" do
   pod "CommonUI", :path => "LocalPods/CommonUI"
   pod "Services", :path => "LocalPods/Services"
 
-  pod "Orcam", :macro => {
-    :git => "https://github.com/trinhngocthuyen/orcam.git",
-    :branch => "main",
-  }
-  pod "MacroCodableKit", :macro => {
-    :git => "https://github.com/mikhailmaslo/macro-codable-kit",
-    :tag => "0.3.0",
-  }
+
+  prebuild_macro = false
+  if prebuild_macro
+    pod "Orcam", :macro => {
+      :git => "https://github.com/trinhngocthuyen/orcam.git",
+      :branch => "main",
+    }
+    pod "MacroCodableKit", :macro => {
+      :git => "https://github.com/mikhailmaslo/macro-codable-kit",
+      :tag => "0.3.0",
+    }
+  else
+    spm_pkg "Orcam", :git => "https://github.com/trinhngocthuyen/orcam.git", :branch => "main"
```

Comment:
I think macro package is just like any other package which can be integrated with `spm_pkg`. So we don't necessarily need to have if/else here :)
If we do wanna showcase that macro can be integrated in both ways, we can consider having one of them with `pod ..., :macro => ...` and the other one with `spm_pkg`.
```ruby
pod "Orcam", :macro => {
  :git => "https://github.com/trinhngocthuyen/orcam.git",
  :branch => "main",
}

spm_pkg "Orcam", :git => "https://github.com/trinhngocthuyen/orcam.git", :branch => "main"
```

Reply:
The reason I integrated both of them with `pod ..., :macro => ...` is to reduce the integration test time (on Github Actions).
It takes quite some time to build swift-syntax packages LOL.

Reply:
I will revert these changes to take the prebuild action